src/main/resources/assets/design-system/margin-padding.scss

```diff
@@ -55,15 +55,15 @@
   margin-top: var(--sci-margin-4);
 }
 
-.sci-mr-0.5, .sci-me-0.5 {
+.sci-mr-0.5, .sci-me-0\.5 {
```

Comment:
Missing escape for `.sci-mr-0.5`? Others seem to be missing, especially when more than one class is listed